src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/LegacyPbaNumbersRetrieveByUserEmailTest.java

```diff
@@ -43,8 +43,8 @@ public void can_retrieve_payment_numbers_by_user_email() {
                 .build());
 
         Map<String, Object> emailResponse = professionalApiClient.retrieveLegacyPbaNumbersByUserEmail(email.toLowerCase());
-        assertThat(emailResponse.get("payment_accounts")).asList().isNotEmpty();
-        assertThat(emailResponse.get("payment_accounts")).asList().contains(pbaNumber.toUpperCase());
+        assertThat(emailResponse.get("paymentAccounts")).asList().isNotEmpty();

```

Comment:
it is good to follow Sonar comment here for not having variable name in camel context , but since FR is already consuming the response field "payment_accounts" and they will fail to get response field, so revert back and make it as existing

Reply:
ok makes sense